### PR TITLE
 Change implementation from .NET framework to .NET core 2.1

### PR DESCRIPTION
**Pull request by umairdojki**

### CODE REVIEW

**Comment by neilkennedy** [comment]
@umairdojki can you rename the `QuestionServiceWebApi` project to be consistent with the other two. Something like `PairingTest.WebApi`. (Or rename the other two projects )

File: src/QuestionServiceWebApi/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```
Comment (neilkennedy) [inline src/QuestionServiceWebApi/appsettings.json]: Will this allow CORs requests from the web project in javascript?

**Review by neilkennedy — CHANGES_REQUESTED** [review]
See comments.

**Comment by yetanotherchris** [comment]
It's nice to see your progress but unsubscribe me from this bloody thread! Argh

File: src/QuestionServiceWebApi/appsettings.json (hunk 1 above)
Comment (umairdojki) [reply src/QuestionServiceWebApi/appsettings.json]: This was a default setting. It doesn't seem to have any effect on whether a CORS request is allowed or not. To enable CORS in the app, we will have to make some changes in Startup.cs, and in the Controller. See here: https://docs.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-2.1 (Did follow these and managed to get it working).

**Review by neilkennedy — APPROVED** [review]

**Pull request merged**